### PR TITLE
Fixes #166: Provide way to access active extension data and metadata

### PR DESCRIPTION
**Pull request by nicolaferraro**
This is needed by artifact generation

### CODE REVIEW

**Comment by nicolaferraro** [comment]
retest this please

**Comment by lburgazzoli** [comment]
retest this please

**Comment by lburgazzoli** [comment]
retest this please

File: controllers/src/main/java/io/syndesis/controllers/extension/ExtensionDataProvider.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    public InputStream getExtensionBinaryFile(String extensionId) {
+        String id = getInstalledExtensionId(extensionId);
+        return fileStore.read("/extensions/" + id);
+    }
+
+    // ==========================================================
+
+    @Nonnull
+    private String getInstalledExtensionId(String extensionId) {
```
Comment (lburgazzoli) [inline controllers/src/main/java/io/syndesis/controllers/extension/ExtensionDataProvider.java]: The name is a little bit misleading as it seems it returns the extensionid by giving an extensionid, maybe something like getInstalledPhysicalId or whatever make it more clear.

Comment (nicolaferraro) [reply controllers/src/main/java/io/syndesis/controllers/extension/ExtensionDataProvider.java]: Right!

**Pull request merged**